Source/WebCore/dom/Observable.idl

```diff
@@ -32,4 +32,6 @@ typedef (SubscriptionObserverCallback or SubscriptionObserver) ObserverUnion;
 interface Observable {
   [RaisesException] constructor(SubscriberCallback callback);
```

Comment:
I do not think this needs `[RaisesException]` after we changed the `create` function to never return an exception.